spec/rollout_spec.rb

```diff
@@ -305,6 +305,21 @@
 
   end
 
+  describe '#clear_collections_from_feature' do
+    before do
+      @rollout.define_id_collection(:will_remove,     [200, 205])
+      @rollout.define_id_collection(:will_remove_too, [300, 305])
+      @rollout.add_collection_to_feature(:will_remove,    :my_feature)
```

Comment:
if u are gonna add extra spaces for this to be aligned, u are missing one space, if not, remove extra spaces :P